app/javascript/mastodon/features/onboarding/follows.tsx

```diff
@@ -170,7 +170,7 @@ export const Follows: React.FC<{
         }
       >
         {displayedAccountIds.map((accountId) => (
-          <Account id={accountId} key={accountId} withBio />
+          <Account id={accountId} key={accountId} withBio hideMenuOnMobile />
```

Comment:
Maybe here we just want to remove the dropdown at all sizes?

Reply:
Done, changed this to a `withMenu` prop that has a default value of true and is set to false here